var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -197,7 +197,8 @@ class Ascent(CMakePackage, CudaPackage):
     # VTK-m
     #######################
     depends_on("vtk-m@2.0:", when="@0.9.2: +vtkh")
-    depends_on("vtk-m@1.9:1.9", when="@0.9.0: +vtkh")
+    depends_on("vtk-m@1.9", when="@0.9.0:0.9.1 +vtkh")
+    depends_on("vtk-m +doubleprecision ~64bitids", when="+vtkh ^vtk-m")
```

Comment:
Is this correct @cyrush?

Reply:
Looks like there should be an upperbound, cause 0.9.2 fails with vtk-m 2.1 due to missing include:

```
src/libs/rover/ray_generators/camera_generator.cpp:77:20: error: 'CanvasRayTracer' is not a member of 'vtkm::rendering'
```

Reply:
Added an upperbound too (found https://github.com/Alpine-DAV/ascent/commit/e52b7bb8c9fd131f2fd49edf58037cc5ef77a166)